front/components/SongRow.tsx

```diff
@@ -0,0 +1,72 @@
+import { HStack, IconButton, Image, Text } from "native-base";
+import Song, { SongWithArtist } from "../models/Song";
+import RowCustom from "./RowCustom";
+import TextButton from "./TextButton";
+import { MaterialIcons } from "@expo/vector-icons";
+
+type SongRowProps = {
+	song: Song | SongWithArtist; // TODO: remove Song
+	onPress: () => void;
+};
+
+const SongRow = ({ song, onPress }: SongRowProps) => {
+	return (
+		<RowCustom width={'100%'}>
```

Comment:
This entier component has already been done. Please see `SongRow`

Reply:
wdym ? SongRow.tsx is indeed the only file where this component has been created and exported